### PR TITLE
Adding tests and workflows

### PR DESCRIPTION
**Pull request by mmatera**
This is going to fail for a while until the documentation be complete.

### CODE REVIEW

File: .github/workflows/ubuntu.yml [hunk 1]
```diff
@@ -1,40 +1,36 @@
...
         (cd src/mathics3 && bash ./admin-tools/make-op-tables.sh)
         # python -m pip install Mathics3[full]
-    - name: Install pymathics.natlang
+        python -m pip install -e .
+    - name: install pymathics natlang
       run: |
-        pip install -e .
-    - name: Test Mathics3
+        make develop
+    - name: Test Mathics
```
[the remark this answers is not in the document]
Comment (mmatera) [reply .github/workflows/ubuntu.yml:34]: I copied it from pymathics.graph

[the remark this answers is not in the document]
Comment (mmatera) [reply .github/workflows/ubuntu.yml:34]: In any case, this is going to take a while...

[the remark this answers is not in the document]
Comment (rocky) [reply .github/workflows/ubuntu.yml:34]: > I copied it from pymathics.graph

Ok. It has been adjusted in pymathics.graph then.

**Comment by mmatera** [comment]
@rocky,
Now, doctest pass,  and natlang docs are built in pdf. Maybe is a good point to review this. Then,  I am going over the documentation urls and summaries.

**Comment by rocky** [comment]
LGTM - merging in now. I will make a pass over summaries. This is something I am more naturally suited to than you. (Know strengths weaknesses).   And this will go into this week's dockerhub image update before the 6.0.0 release.

It would be a great help if you would reduce the missing label warnings and other warnings that happen in mathics-core which building the docs.  Thanks.

**Pull request merged**